scripts/test/Muon/fitting_tab_widget/fitting_tab_presenter_test.py

```diff
@@ -99,21 +101,44 @@ def test_for_single_fit_mode_when_display_workspace_changes_updates_fitting_brow
 
         self.assertEqual(self.view.function_browser.getDatasetNames(), ['Input Workspace Name'])
 
-    def test_fit_clicked_with_simultaneous_selected(self):
+    def test_fit_clicked_with_simultaneous_selected_and_no_globals(self):
         self.view.simul_fit_radio.toggle()
         self.presenter.selected_data = ['Input Workspace Name_1', 'Input Workspace Name 2']
         self.view.function_browser.setFunction('name=GausOsc,A=0.2,Sigma=0.2,Frequency=0.1,Phi=0')
+        self.presenter.model.do_simultaneous_fit.return_value = (self.view.function_browser.getGlobalFunction(),
+                                                                 'Fit Suceeded', 0.5)
 
         self.view.fit_button.clicked.emit(True)
         wait_for_thread(self.presenter.calculation_thread)
 
-        call_args_dict = self.presenter.model.do_simultaneous_fit.call_args[0][0]
+        simultaneous_call_args = self.presenter.model.do_simultaneous_fit.call_args
+        call_args_dict = simultaneous_call_args[0][0]
 
         self.assertEqual(call_args_dict['InputWorkspace'], ['Input Workspace Name_1', 'Input Workspace Name 2'])
         self.assertEqual(call_args_dict['Minimizer'], 'Levenberg-Marquardt')
         self.assertEqual(call_args_dict['StartX'], [0.0, 0.0])
         self.assertEqual(call_args_dict['EndX'], [15.0, 15.0])
 
+        call_args_globals = simultaneous_call_args[0][1]
+        self.assertEqual(call_args_globals, [])
+
+    def test_fit_clicked_with_simultaneous_selected_with_global_parameters(self):
+        self.view.simul_fit_radio.toggle()
+        self.presenter.selected_data = ['Input Workspace Name_1', 'Input Workspace Name 2']
+        self.view.function_browser.setFunction('name=GausOsc,A=0.2,Sigma=0.2,Frequency=0.1,Phi=0')
+        self.view.function_browser.setGlobalParameters(['A'])
+        self.presenter.model.do_simultaneous_fit.return_value = (self.view.function_browser.getGlobalFunction(),
+                                                                 'Fit Suceeded', 0.5)
+
+        self.view.fit_button.clicked.emit(True)
+        wait_for_thread(self.presenter.calculation_thread)
+
+        simultaneous_call_args = self.presenter.model.do_simultaneous_fit.call_args
+
+        # above tests output with no globals set
```

Comment:
```suggestion
        # above tests output with global set
```

Reply:
I did mean no globals here as I was referring to the above test method but on second thoughts I think it might be better to test the full output here too rather than relying on knowing the implementation details.

Reply:
ah ok